index/miner/meta.go

```diff
@@ -2,11 +2,12 @@ package miner
 
 import (
```

Comment:
Mostly enabling some capped parallelization to discover metadata about miners. Capped at 100 max at a time, could be played bit.